### DISCUSSION TITLE
MQTT?

### DISCUSSION THREAD

**Discussion by glynhudson** [flobz/psa_car_controller#219] [Ideas]
It would be nice if an option was available to publish the vehicle data to MQTT, this would allow easy integration with home automation software and some EV charging stations e.g OpenEVSE

**Comment by flobz** [comment]
I won't develop it but if someone do it, I will be happy to add it to the app.  You can also create a two line program that get data from http and send it via mqtt. Nodered can do it also if your aren't good in programming.

**Reply by pdhoogh** [reply]
@flobz Do you mean the following?

A separate python program requests data using http://localhost:5000/get_vehicleinfo/YOURVIN?from_cache=1. The python program parses the response and publishes the required elements to MQTT.

That seems like more than two lines.
